### PR TITLE
Add-related-content-to-say-brother

### PR DESCRIPTION
**Pull request by caseyedavis12**

### CODE REVIEW

File: app/views/collections/say-brother.md [hunk 1]
```diff
@@ -22,3 +22,11 @@ conversations with distinct voices from the African American community.
 ## Records
 
 [](http://localhost:3000/catalog?f[special_collection_tags][]=sb_record)
+
+## Related Content
+
+[Basic Black](http://wgbh.org/basicblack)
```
Comment (mccalluc) [inline app/views/collections/say-brother.md]: @caseyedavis12 : This redirects to plain wgbh.org. Do you have a replacement?

Comment (caseyedavis12) [inline app/views/collections/say-brother.md]: @mccalluc: Can you add www. before wgbh.org? That will fix it.


________________________________

**Comment by mccalluc** [comment]
I would worry a little about long term plans for the main.wgbh.org pages. The whole Say Brother collection is duplicated there, except on that site the videos, for me at least, do not play. I also don't know who controls that server and is responsible for its maintenance.

@caseyedavis12 : I won't file an issue, maybe there's a reason for the duplication, but you're free to.

**Comment by caseyedavis12** [comment]
@mccalluc: Basic Black (formerly Say Brother) is a current production at WGBH. We will know if they decide to end production and can at that time remove the link. Hopefully they will continue producing new content well into the future.

**Pull request merged**